Cargo.toml

```diff
@@ -46,8 +46,12 @@ filetime = "0.2.10"
 
 [features]
 default = ["native-roots"]
+# Use Rustls with native roots
```

Comment:
For me, the comment doesn't provide any more information over the line below, I am in favor of removing it again